embed-code-go/fragmentation/fragment_file.go

```diff
@@ -95,8 +95,10 @@ func (fragmentFile FragmentFile) Content() []string {
 	isPathFileExits, err := IsFileExists(path)
 	if isPathFileExits {
 		return ReadLines(path)
-	} else {
+	} else if err != nil {
 		panic(err)
```

Comment:
This differs from your ussual `err` handling in most cases you have something like: 
```
isPathFileExits, err := IsFileExists(path)
if err != nil {
		panic(err)
}
// work with file
```